_includes/samples/VRF/VRFD20.sol

```diff
@@ -9,6 +9,12 @@ import "@chainlink/contracts/src/v0.8/ConfirmedOwner.sol";
  * of a 20 sided die
  * @dev This is only an example implementation and not necessarily suitable for mainnet.
  */
+
+/**
+ * Request testnet LINK and ETH here: https://faucets.chain.link/
+ * Find information on LINK Token Contracts here: https://docs.chain.link/docs/link-token-contracts/
```

Comment:
```suggestion
 * Request testnet LINK and ETH here: https://faucets.chain.link/
 * Find information on LINK Token Contracts and get the latest ETH and LINK faucets here: https://docs.chain.link/docs/link-token-contracts/
```

Reply:
Sounds good; for the sake of brevity and accuracy the first option is better; I'll go ahead and change the comment to refer to https://docs.chain.link/docs/link-token-contracts/ and also put on my todo list to add info about the latest rinkeby, bsc testnet, fantom testnet, etc faucets into the faucets page